projects/fmcdaq2/src/app/fmcdaq2.c

```diff
@@ -79,12 +79,11 @@
 #include "xilinx_uart.h"
 #endif
 
-#ifdef JESD_FSM_ON
 #include "no_os_print_log.h"
 #include "no_os_alloc.h"
 #include "jesd204.h"
 #include "jesd204_clk.h"
-#endif
+
```

Comment:
remove this line
i beliece this commit should be all red, no green :D

Reply:
Removed

Reply:
actually i take this back, i misread the diff...